File: 程式語言 V2答案檔/C#/202.cs
using System;

public class TQC202
{
    public static void Main(string[] args)
    {
        int score;
        score = int.Parse(Console.ReadLine());
        
        if (score < 0 || score > 100) {
            Console.WriteLine("error");
        }
        else {
            if (score > 60)
                score += 10;
            else
                score += 5;

            Console.WriteLine(score);
        }
    }
}

File: 程式語言 V2答案檔/Go/202.go
package main
import "fmt"

func main(){
	var a int
	fmt.Scan(&a)
	
	if a > 100{
		fmt.Println("error")
		return
	}

	if a > 60{
		a += 10
	}else{
		a += 5
	}

	fmt.Println(a)
}
